stdlib/buffer.mli

```diff
@@ -148,6 +148,14 @@ val add_channel : t -> in_channel -> int -> unit
    characters. In this case, the characters are still added to
    the buffer, so as to avoid loss of data. *)
 
+val add_channel_full : t -> in_channel -> int
+(** [add_channel b ic] reads all the available characters from
```

Comment:
This should be `add_channel_full`. I would be more terse and define it with reference to `add_channel` - e.g. `[add_channel_full] is as [add_channel] but returns the number of characters read.`

Reply:
The terser explanation you give is not really correct, as `add_channel_full` does not take a limit on the total length to read. I fixed the copy-error.

Reply:
Sorry - I realised this morning. It turns out reviewing on one's phone limits the visible text...